### PR TITLE
changes made to support openSUSE

### PR DESCRIPTION
**Pull request by jassigill2000**

### CODE REVIEW

**Comment by jassigill2000** [comment]
needs to be merged for CI to work properly

File: irods_python_ci_utilities/copied_from_ansible.py [hunk 1]
```diff
@@ -44,7 +44,8 @@ def get_distribution():
                     distribution = 'OtherLinux'
         except:
             # FIXME: MethodMissing, I assume?
```
Comment (trel) [inline irods_python_ci_utilities/copied_from_ansible.py]: remove comment if this fixes it

File: irods_python_ci_utilities/copied_from_ansible.py [hunk 2]
```diff
@@ -59,7 +60,8 @@ def get_distribution_version():
                 distribution_version = platform.linux_distribution(supported_dists=['system'])[1]
         except:
             # FIXME: MethodMissing, I assume?
```
Comment (trel) [inline irods_python_ci_utilities/copied_from_ansible.py]: remove comment if this fixes it

File: irods_python_ci_utilities/irods_python_ci_utilities.py [hunk 3]
```diff
@@ -55,30 +55,31 @@ def subprocess_get_output(*args, **kwargs):
...
     }
     try:
-        dispatch_map[get_distribution()](packages)
+        return dispatch_map[get_distribution()](packages)
     except KeyError:
         raise_not_implemented_for_distribution()
 
 def install_os_packages_from_files_apt(files):
-    '''files are installed individually in the order supplied, so inter-file dependencies must be handled by the caller'''
+    #'''files are installed individually in the order supplied, so inter-file dependencies must be handled by the caller'''
```
Comment (trel) [inline irods_python_ci_utilities/irods_python_ci_utilities.py]: what does this # do?

File: irods_python_ci_utilities/irods_python_ci_utilities.py [hunk 4]
```diff
@@ -89,14 +90,16 @@ def install_os_packages_from_files_yum(files):
     subprocess_get_output(args, check_rc=True)
 
 def install_os_packages_from_files_zypper(files):
-    install_os_packages_zypper(files)
+    args = ['sudo', 'zypper', '--no-gpg-checks', '--non-interactive', 'install'] + list(files)
```
Comment (trel) [inline irods_python_ci_utilities/irods_python_ci_utilities.py:93]: do we have a good reason to disable the encryption?

Comment (jassigill2000) [reply irods_python_ci_utilities/irods_python_ci_utilities.py:93]: if the files are in a file system like a volume mount then there are no keys associated with them.

Comment (trel) [reply irods_python_ci_utilities/irods_python_ci_utilities.py:93]: ah, so zypper can just install files from a local directory.

File: irods_python_ci_utilities/irods_python_ci_utilities.py (hunk 3 above)
Comment (jassigill2000) [reply irods_python_ci_utilities/irods_python_ci_utilities.py]: I was under the impression that string was supposed to be a comment.

Comment (trel) [reply irods_python_ci_utilities/irods_python_ci_utilities.py]: sure, so no need for the triple quotes, right?

**Pull request merged**